xml/System.Net/SecurityProtocolType.xml

```diff
@@ -29,16 +29,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This enumeration defines permissible values for the <xref:System.Net.ServicePointManager.SecurityProtocol%2A> property and specifies the security protocols that are used by instances of the <xref:System.Net.Security.SslStream> class.  
-  
-   
-  
-## Examples  
- The following code example demonstrates creating a <xref:System.Net.Sockets.TcpClient> that uses the <xref:System.Net.Security.SslStream> class to communicate with a server.  
-  
- [!code-cpp[NclSslClientSync#0](~/samples/snippets/cpp/VS_Snippets_Remoting/NclSslClientSync/CPP/clientsync.cpp#0)]
- [!code-csharp[NclSslClientSync#0](~/samples/snippets/csharp/VS_Snippets_Remoting/NclSslClientSync/CS/clientsync.cs#0)]  
-  
+ This enumeration defines allowed values for the <xref:System.Net.ServicePointManager.SecurityProtocol%2A>, which is used by <xref:System.Net.Http.HttpClient> when connecting to a server.  
```

Comment:
> which is used by System.Net.Http.HttpClient

This is not actually correct.  

The real text should say something like:
"The ServicePointManager.SecurityProtocolType is used by the various HTTP APIs in .NET Framework which include System.Net.WebClient, System.Net.HttpWebRequest and System.Net.Http.HttpClient. ...